docs/operator/detailed_guide/README.md

```diff
@@ -0,0 +1,286 @@
+# Polymesh Operator - Detailed Guide
+
+### **Strictly Confidential**
```

Comment:
This is a public repo then what is a need for  **Strictly Confidential**

Reply:
I copied this section verbatim from the document shared on Teams.  I will reach out to Nick to review the legalese.